examples/example-phase1/src/app/(pages)/settings/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,24 @@
+// Services
+import { getBlogBySlug } from "@/services/blog";
+
+interface BlogDetailPageProps {
+  params: {
+    slug: string
+  }
+}
+
+const BlogDetail = async ({ params }: BlogDetailPageProps) => {
+  const slug = params?.slug;
+
+  const blog = await getBlogBySlug(slug as string);
```

Comment:
Prevent to use type assertion here

Reply:
BTW, how can the server get information of an item by slug ??

Reply:
Keep in mind, pls check null or undefine first. Don't use type assertion, it's so dangerouse